src/utils/consistency/db.rs

```diff
@@ -30,23 +34,19 @@ pub(super) fn load(conn: &mut postgres::Client, config: &Config) -> Result<Crate
                  releases.id IS NULL
              )
          ) AS inp
-         ORDER BY name, version",
-        iter::once(config.build_attempts as i32),
-    )?;
+         ORDER BY name, version"#,
+        config.build_attempts as i32,
+    )
+    .fetch_all(conn)
```

Comment:
this has higher memory usage than the implementation before, since `fetch_all` loads everything into a `Vec`. 

But since I still wanted to use `Itertools::chunk_by` below, I thought we bite the bullet, since we have enough memory. 

( I'll optimize it later)